spec/mailers/previews/notify_mailer_preview.rb

```diff
@@ -39,7 +39,7 @@ def channel_invite_email
   end
 
   def tag_moderator_confirmation_email
-    NotifyMailer.tag_moderator_confirmation_email(User.first, Tag.find(1))
+    NotifyMailer.tag_moderator_confirmation_email(User.last, Tag.find(1), nil)
```

Comment:
Just curious what the reasoning is for changing this to `User.last`.

Reply:
I changed it to User.last just coz I was testing with my user and it happened to be the last user. It really wouldn't make a difference, but I changed it back to User.first coz I shouldn't have committed that. Thanks!